README.md

```diff
@@ -1,129 +1,170 @@
-# Controls
-This repository contains all code related to Longhorn Racing Solar's Controls System
+# LHRS Controls
+
+This repository contains all code related to Longhorn Racing Solar's Controls System. The Controls System is responsible for driver interactions, motor control, and lighting systems control.
+
+## Table of Contents
+
+1. [About the Project](#about-the-project)
+1. [Project Status](#project-status)
+1. [Getting Started](#getting-started)
+	1. [Workspace Setup](#workspace-setup)
+	1. [Build System](#build-system)
+    1. [Running Tests](#running-tests)
+    1. [Formatting](#formatting)
+    1. [Usage](#usage)
+1. [Release Process](#release-process)
+	1. [Versioning](#versioning)
+1. [Contributing](#contributing)
+    1. [Branch Convention](#branch-convention)
+    1. [Review Process](#review-process)
+1. [Documentation](#documentation)
+1. [License](#license)
+1. [Authors](#authors)
+1. [Acknowledgments](#acknowledgements)
+
+## About the Project
+
+This repository contains all code related to [Longhorn Racing Solar](https://www.longhornracing.org/solar-vehicle-team)'s Controls System, affiliated with **UT Austin**. The Controls System is responsible for driver interactions, motor control, and lighting systems control. The system is built on [custom hardware](https://github.com/lhr-solar/Controls-LeaderPCB) and an ARM Cortex-M4 Processor (STM32F413).
+
+For more information, see our [documentation](https://controls-docs.readthedocs.io/en/latest/).
+
+**[Back to top](#table-of-contents)**
+
+## Project Status
+
+The Controls System is currently on release **M2.0** for the 2022-24 build cycle, to compete in FSGP 2024. See our [Versioning](#versioning).
+
+**[Back to top](#table-of-contents)**
+
+## Getting Started
+
+The preferred development environment for the system is VSCode on a Linux machine. Options for developers on Windows environments include WSL or a Virtual Machine. Instructions provided here will assume that you have a working Linux environment with terminal control.
+
+### Workspace Setup
+
```

Comment:
Did you want to include directions on recommended linux setup? I think most people don't know so maybe doing a wsl installation thing would be nice. It can also just point to other websites if you don't want to write them yourself.

Reply:
I'll link it to other websites.